addons/l10n_tr/demo/account_demo.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+import logging
+from datetime import date
+
+from odoo import api, models, Command
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    @api.model
+    def _get_demo_data(self):
+        ref = self.env.ref
+
+        # Do not load generic demo data on these companies
+        tr_demo_companies = (
+            ref('l10n_tr.demo_company_tr', raise_if_not_found=False),
+        )
+        if self.env.company in tr_demo_companies:
+            yield self._get_demo_data_move_tr()
+
+        else:
+            for model, data in super()._get_demo_data():
+                yield model, data
+
+    @api.model
+    def _post_create_demo_data(self, created):
+        cid = self.env.company.id
+        ref = self.env.ref
+
+        # Do not load generic demo data on these companies
+        tr_demo_companies = (
+            ref('l10n_tr.demo_company_tr', raise_if_not_found=False),
+        )
+        if self.env.company not in tr_demo_companies:
+            return super()._post_create_demo_data(created)
+
+        if created._name == 'account.move':
+            created = created.with_context(check_move_validity=False)
+            # We need to recompute some onchanges. Invoice lines VS journal items are already
+            # synchronized in the create, but the onchange were not applied on the invoice lines.
+            # This will remove the manually set Fiscal Position on the move because there is a
+            # Domestic (standard) FP and an Export ones that one of them will apply. In reality
+            # the user first sets the partner, then the FP and finally the lines, we have to do the same
```

Comment:
I was planing to add the bank statements later, so I did not remove the comments. I kept the original comments for being clear. 
I couldn't use the original function because it was adding an invoice for OCR and I didn't have one